MinStack.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
Kindly try to submit all the questions together instead of submitting them separately.